app/jobs/recaptcha_annotate_job.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class RecaptchaAnnotateJob < ApplicationJob
+  include GoodJob::ActiveJobExtensions::Concurrency
+
+  BASE_ENDPOINT = 'https://recaptchaenterprise.googleapis.com/v1'
+
+  queue_as :low
+
+  good_job_control_concurrency_with(
+    enqueue_limit: 1,
+    key: -> { "#{self.class.name}-#{queue_name}-#{arguments.last[:assessment_id]}" },
+  )
+
+  def perform(assessment_id:, reason: nil, annotation: nil)
```

Comment:
Noticed this in `RecapatchaAnnotation` as well - why do we allow this to be `nil` if we are always supplying a reason?

Reply:
The thought process here was that both are optional arguments in code because both are [optional parameters of the annotation API](https://cloud.google.com/recaptcha/docs/annotate-assessment).

I don't have a strong feeling to not make it required based on current usage, though I could imagine that might cause confusion down the line if someone didn't realize they didn't _have to_ provide a reason.

Reply:
Hmm, one of the values in https://cloud.google.com/recaptcha/docs/reference/rest/v1/projects.assessments/annotate#reason is `REASON_UNSPECIFIED Unspecified reason. Do not use.` (sic)